lms/djangoapps/discussion/views.py

```diff
@@ -332,20 +329,18 @@ def single_thread(request, course_key, discussion_id, thread_id):
                 request.user,
                 user_info=user_info
             )
+
         content = utils.prepare_content(thread.to_dict(), course_key, is_staff)
         with newrelic.agent.FunctionTrace(nr_transaction, "add_courseware_context"):
             add_courseware_context([content], course, request.user)
+
         return utils.JsonResponse({
             'content': content,
             'annotated_content_info': annotated_content_info,
         })
-
     else:
-        try:
-            threads, query_params = get_threads(request, course, user_info)
-        except ValueError:
-            return HttpResponseBadRequest("Invalid group_id")
-        threads.append(thread.to_dict())
+        # We only care about providing enough context to render the view for this single thread.
```

Comment:
Does it make sense to note that this only gets used during a single thread view page refresh?